receiver/datadogreceiver/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License. language governing permissions and
+// limitations under the License.
+
+package datadogreceiver
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/confighttp"
+)
+
+type Config struct {
+	config.ReceiverSettings       `mapstructure:",squash"`
+	confighttp.HTTPServerSettings `mapstructure:",squash"`
+	ReadTimeout                   time.Duration `mapstructure:"read_timeout"` // The read timeout of the http server
```

Comment:
this should be
```suggestion
        // ReadTimeout of the http server
	ReadTimeout                   time.Duration `mapstructure:"read_timeout"`
```

so that it is rendered correctly in the docs